README.md

```diff
@@ -131,6 +174,119 @@ Note that we are replacing the test job image (`k6-prometheus:latest`), passing
 (`--out prometheus`), and also exposing the ports required for Prometheus to scrape the metrics
 (in this case, that's port `5656`)
 
+You'll also need to create a pod monitor: 
```

Comment:
This only applies if the user is using the Prometheus Operator. Right?

I'm not very familiar with it...

Reply:
Yes, I can update that.